ghcide/src/Development/IDE/Core/Compile.hs

```diff
@@ -902,8 +902,8 @@ indexHieFile se mod_summary srcPath !hash hf = do
             Just pendingHash -> pendingHash /= hash
         unless newerScheduled $ do
           pre optProgressStyle
```

Comment:
This should be removed?

Reply:
Yes. Sorry about that.

Reply:
Fixed. I'm so sorry (i've done some cleanup before sending the MR and as usual, I broke stuff)